raiden/tests/integration/test_events.py

```diff
@@ -198,19 +197,19 @@ def test_query_events(raiden_chain, token_addresses, deposit, settle_timeout, re
         events=ALL_EVENTS,
     )
 
-    channel_id = channel_identifier(app0.raiden.address, app1.raiden.address)
-    assert must_have_event(
+    _event = must_have_event(
         events,
         {
             'event': EVENT_CHANNEL_OPENED,
             'args': {
                 'participant1': to_checksum_address(app0.raiden.address),
                 'participant2': to_checksum_address(app1.raiden.address),
                 'settle_timeout': settle_timeout,
-                'channel_identifier': channel_id,
```

Comment:
how is this fixing it? It just removed the assertion.

Reply:
The `channel_id` can't be (or is very hardly) calculated only with the channel opening parameters, but the current channel open is unique for the pair of peers. This changes asserting the specific `channel_identifier`, to fetching it from the event and using it elsewhere it's needed.